src/foam/mlang/mlang.js

```diff
@@ -764,6 +764,20 @@ getArg2().prepareStatement(stmt);`
         getArg1().authorize(x);
         getArg2().authorize(x);
       `
+    },
+    function arg2ToMQL() {
+      if ( this.arg2 && this.arg2.value && foam.Date.isInstance(this.arg2.value) ) {
```

Comment:
This is when arg2 is a Constant? Couldn't this logic be moved to Constant and we just call toMQL() on arg2?